docs/user_guides/debugging/stepping.rst

```diff
@@ -16,7 +16,7 @@ Example with a nested function ``numba_dppy/examples/debug/simple_dppy_func.py``
 .. note::
 
     Known issues:
-        - Debug of the first line of the kernel and functions works out twice. See :ref:`single_stepping`.
+        - The first line of the kernel and functions is debugged twice. See :ref:`single_stepping`.
```

Comment:
reword "is debugged twice". More importantly, do we know what causes the problem? Worth putting an explanation here.